examples/process-compose/devenv.nix

```diff
@@ -7,16 +7,28 @@
 
   postgres.enable = true;
 
+  languages.ruby.enable = true;
+
+  packages = [ pkgs.imagemagick6.dev ];
```

Comment:
This is a bit sub-optimal from a usage standpoint.
Multiple output packages are in general a pain to discover.
Maybe extend `devenv search` to include package outputs?

Reply:
We should just call `getOutput` on all the outputs for packages and include them all. I think that's a better default for non-experienced Nixers, while you can still pass `pkg.dev` if you want to reduce the closure size.

Reply:
```
nix-repl> lib.flatten (builtins.map (package: builtins.map lib.getOutput package.outputs) [git vim])
[ «lambda @ /home/domen/dev/nixpkgs/lib/attrsets.nix:825:23» «lambda @ /home/domen/dev/nixpkgs/lib/attrsets.nix:825:23» «lambda @ /home/domen/dev/nixpkgs/lib/attrsets.nix:825:23» «lambda @ /home/domen/dev/nixpkgs/lib/attrsets.nix:825:23» ]
```